CorsixTH/Lua/world.lua

```diff
@@ -74,6 +74,16 @@ function World:World(app)
   self.rooms = {} -- List that can have gaps when a room is deleted, so use pairs to iterate.
   self.entity_map = EntityMap(self.map)
 
+  -- All information relating to the next or current earthquake, nil if
+  -- there is no scheduled earthquake.
+  -- Contains the following fields:
+  -- active (boolean) Whether the earthquake is currently happening.
+  -- start_month (integer) The month the earthquake is triggered.
+  -- start_day (integer) The day of the month the earthquake is triggered.
+  -- stop_day (integer) the day of the month the earthquake ends.
+  -- size (integer) The amount of damage the earthquake causes (1-9).
+  self.next_earthquake = nil
```

Comment:
Agreed to always set ```{ active = false }``` but it's not enough to test that because an earthquake scheduled for the future is also ```active = false``` (but with a start_month and start_day).